website/docs/r/cloud_aws_integrations.html.markdown

```diff
@@ -368,11 +364,84 @@ All other arguments are dependent on the services to be integrated, which have b
 
 </details>
 
+## Argument Reference
+
+-> **WARNING:** Starting with [v3.27.2](https://registry.terraform.io/providers/newrelic/newrelic/3.27.2) of the New Relic Terraform Provider, updating the `linked_account_id` of a `newrelic_cloud_aws_integrations` resource that has been applied would **force a replacement** of the resource (destruction of the resource, followed by the creation of a new resource). Please carefully review the output of `terraform plan`, which would clearly indicate a replacement of this resource, before performing a `terraform apply`.
+
+* `account_id` - (Optional) The New Relic account ID to operate on.  This allows the user to override the `account_id` attribute set on the provider. Defaults to the environment variable `NEW_RELIC_ACCOUNT_ID`.
+* `linked_account_id` - (Required) The ID of the linked AWS account in New Relic.
+
+
+The following arguments/integration blocks are intended to be used with a minimum metrics polling interval of 300 seconds
```

Comment:
```suggestion
The following arguments/integration blocks are intended to be used with a minimum `metrics_polling_interval` of 300 seconds.
```

(thanks for correcting this, but this is still not exactly in the format I've mentioned - just trying to make sure this sentence has appropriate punctuation)

Reply:
updated the document, kindly review